LICENSE

```diff
@@ -1,6 +1,5 @@
-Flynn® is a trademark of Prime Directive, Inc.
-
-Copyright (c) 2013-2015 Prime Directive, Inc. All rights reserved.
+Copyright (c) 2013-2015 Prime Directive, Inc.
```

Comment:
Let's bump the year to 2018 while we're here.

Reply:
Good call!